hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice/src/main/java/org/apache/hadoop/yarn/server/timeline/RollingLevelDBTimelineStore.java

```diff
@@ -18,10 +18,10 @@
 
 package org.apache.hadoop.yarn.server.timeline;
 
+
```

Comment:
Can we delete this line? It is not related.

Reply:
yes